File: KarateKlub/KarateKlub.Api/Models/ClanarinePregledVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace KarateKlub.Api.Models
{
    public class ClanarinePregledVM
    {
        public class Row {
            public int id;
            public string naziv;
            public DateTime datumOd;
            public DateTime datumDo;
        }
        public List<Row> rows;
    }
}

File: Android aplikacija/KarateKlub/app/src/main/java/com/example/x/karateklub/area_blagajnik/model/ClanarinePregledVM.java
package com.example.x.karateklub.area_blagajnik.model;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

public class ClanarinePregledVM implements Serializable {
    public static class Row implements  Serializable{

        public int id;
        public String naziv;
        public Date datumOd;
        public Date datumDo;
    }
    public List<Row> rows;
}
